config/ranks/military.txt

```diff
@@ -1,9 +1,9 @@
 Captain=CPT
-Executive Officer=LT
-Head of Security=WO
-Chief Engineer=WO
-Research Director=WO
-Chief Medical Officer=WO
+Executive Officer=CDR
+Head of Security=LT
+Chief Engineer=LTCDR
```

Comment:
WO specialty: Engineering/Nuclear Power Technician
This isn't even the right abbreviation for lieutenant commander.